### ISSUE TITLE
Perl Warn : Use of uninit value in SpadsConf.pm

### ISSUE THREAD

**Issue by FabriceFABS** [Yaribz/SPADS#38]
Yop !

Today, I restarted autohost [ACE]Ticots (remind : ustable), and I received an error in the console at the end :

```
NOTICE   - [SPADS] Initializing SPADS 0.12.2a
NOTICE   - [SPADS] Loading Spring archives using unitsync library version 104 ...
WARNING  - [SPADS] Spring archives loading process took 2 minutes and 32 seconds
NOTICE   - [SPADS] Spring server mode: dedicated
NOTICE   - [SimpleEvent] Event loop initialized using internal model
NOTICE   - [SPADS] <AutoSpec> Plugin loaded (version 0.1)
NOTICE   - [SPADS] <CustomAliases> Plugin loaded (version 0.1): 77 aliases configured
NOTICE   - [SPADS] <DownloadArchives> Plugin loaded (version 0.1)
NOTICE   - [SPADS] <Resign> Plugin loaded (version 0.2)
NOTICE   - [SPADS] <InGameMute> Plugin loaded (version 0.4)
NOTICE   - [SPADS] <Poke> Plugin loaded (version 0.2)
NOTICE   - [SPADS] <PreventVoteExploit> Plugin loaded (version 0.4)
NOTICE   - [SimpleEvent] Starting event loop...
NOTICE   - [SPADS] TrueSkill service available
ERROR    - [SPADS] PERL WARNING: Use of uninitialized value in string eq at /home/.../SpadsConf.pm line 2235.
```
If you want me publishing extra logs, plz remind me how to enable, I will post it back as soon as I can.

**Comment by FabriceFABS** [comment]
Same on [ACE]Pirateur, but add this extra-line after the error mentioned above :
`ERROR    - [SPADS] PERL WARNING: Use of uninitialized value $ts in subtraction (-) at /home/.../SpadsConf.pm line 1380.`

**Comment by Yaribz** [comment]
It seems you just copy-pasted the standard output of the SPADS console, can you provide the corresponding parts of the log file instead (spads.log) ?

(the spads.log file is basically always needed when you report an error, the standard output doesn't contain all the information)

**Comment by Yaribz** [comment]
Thank you, yet another case of inconsistent data sent by lobby server...
I will make SPADS more resilient when I have some time.

**Comment by Yaribz** [comment]
Should be fixed in SPADS 0.12.3, please reopen if it happens again.
